.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -31,8 +31,8 @@ body:
       label: Reproducing the error
       description: Are you able to reproduce this issue on the [Auro docsite](https://auro.alaskaair.com/)?
       options:
-        - Yes I can
-        - No I can't
+        - This issue is reproducible on the Auro docsite
+        - This issue cannot be reproduced on the Auro docsite
```

Comment:
This isn't a `fix` commit.

Reply:
You are correct and I would like to update this, but this is where broken tests and pre-commit hooks bite us in the rear. 

```
/Users/dalesande/src/auro/WC-Generator/bin/generate.js:7
  throw err;
  ^

SyntaxError: /Users/dalesande/src/auro/WC-Generator/template/package.json: Unexpected token } in JSON at position 912
    at parse (<anonymous>)
    at Module._extensions..json (node:internal/modules/cjs/loader:1322:39)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at getVersionData (/Users/dalesande/src/auro/WC-Generator/bin/generate.js:171:63)
    at getReplacements (/Users/dalesande/src/auro/WC-Generator/bin/generate.js:270:29)
    at copyAllFiles (/Users/dalesande/src/auro/WC-Generator/bin/generate.js:325:30)
    at async generateFromTemplate (/Users/dalesande/src/auro/WC-Generator/bin/generate.js:420:5)

Node.js v18.15.0
husky - pre-commit hook exited with code 1 (error)
```

Might just have to let this one slip past the goalie.